### PR TITLE
TRACEFOSS-381 - added multi select feature to table component

### PR DESCRIPTION
**Pull request by evia-fbedarf**

### CODE REVIEW

File: src/app/modules/shared/components/table/table.component.ts [hunk 1]
```diff
@@ -45,28 +50,56 @@ export class TableComponent {
...
-  updateSortingOfData({ active, direction }: Sort): void {
+  public updateSortingOfData({ active, direction }: Sort): void {
+    this.selection.clear();
     this.sorting = !direction ? null : ([active, direction] as TableHeaderSort);
     this.isDataLoading = true;
     this.configChanged.emit({ page: 0, pageSize: this.pageSize, sorting: this.sorting });
   }
+
+  public toggleSelection(event: MatCheckboxChange, row: unknown): void {
+    if (!event) {
```
Comment (ds-ext-akononenko) [inline src/app/modules/shared/components/table/table.component.ts:94]: do we actually can have situation when there no event?

File: src/app/modules/shared/components/table/table.component.ts [hunk 2]
```diff
@@ -30,6 +32,9 @@ import { TableConfig, TableEventConfig, TableHeaderSort } from '@shared/componen
   styleUrls: ['table.component.scss'],
 })
 export class TableComponent {
+  @ViewChild('SelectColumn') SelectColumn: TemplateRef<any>;
```
Comment (ds-ext-akononenko) [inline src/app/modules/shared/components/table/table.component.ts:35]: 2 questions:
1. Why it's started with upper case? **S**electColumn?
2. Why do you need those references in component? Template should be enough

**Review by ds-ext-akononenko — APPROVED** [review]
In general looks good, 2 minor comments

File: src/app/modules/shared/components/table/table.component.ts (hunk 2 above)
Comment (evia-fbedarf) [reply src/app/modules/shared/components/table/table.component.ts]: Ah, this is a remnant of an old approach. I will remove it.

File: src/app/modules/shared/components/table/table.component.ts (hunk 1 above)
Comment (evia-fbedarf) [reply src/app/modules/shared/components/table/table.component.ts]: Removed it.

**Pull request merged**